source/manual/renew-a-tls-certificate.html.md

```diff
@@ -36,15 +36,9 @@ Credentials for the Fastly dashboard and Zendesk support sites are in the
    .crt file into Puppet hiera data for the relevant environment in the
    [govuk-secrets](https://github.com/alphagov/govuk-secrets) repository.
 10. Deploy Puppet to update the certificate in the relevant environment.
-11. *For staging and integration only:* Use Fastly's
-    [TLS Uploader](https://manage.fastly.com/sslupload/) to upload the newly
-    generated certificate and private key. You need to split the contents of the
-    .crt file into the first certificate (the actual certificate you've just
-    renewed) and everything else (the certificate chain) and upload these as
-    separate files.
-12. *For staging and integration only:* Make a note of the reference ID returned
-    by the TLS Uploader. Open a new ticket with
-    [Fastly support](https://fastly.zendesk.com/hc/en-us) quoting the reference
-    ID and the service ID for the Fastly service the certificate is for
-    (such as for "Staging GOV.UK") and asking them to make the certificate live.
-    This process may take up to a week.
+11. *For staging and integration only:*
+    Go to the Fastly interface and then to Configure -> HTTPS and network
```

Comment:
This line gets merged together with the next one when it's rendered, so this reads "HTTPS and network Go to TLS certificates and upload your new cert." Can fix that by just adding a full stop. Or could make it a sublist if you prefer.
```suggestion
    Go to the Fastly interface and then to Configure -> HTTPS and network.
```

Reply:
Ah right thanks, full stop added.